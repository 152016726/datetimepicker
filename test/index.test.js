import renderer from 'react-test-renderer';
import DatePicker from '../src/index.js';
import React from 'react';

test('renders a DatePicker by default', () => {
  const tree = renderer.create(<DatePicker value={ new Date('08/20/2013') }/>).toJSON();

  expect(tree).toMatchSnapshot();
});
