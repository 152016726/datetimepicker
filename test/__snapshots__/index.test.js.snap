// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a DatePicker by default 1`] = `
<View>
  <RNDateTimePicker
    date={1376949600000}
    mode="time"
    onChange={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "height": 216,
      }
    }
  />
</View>
`;
